### ISSUE TITLE
Swagger CodeGen

### ISSUE THREAD

**Issue by esco** [strongloop/loopback-sdk-ios#37]
Are there any plans on making an SDK generator for iOS similar to [loopback-sdk-angular](https://github.com/strongloop/loopback-sdk-angular)?

**Comment by aaresta** [comment]
+10 for that

**Comment by hideya** [comment]
I'm thinking to work on this after updating the model hierarchy to 2.0: https://github.com/strongloop/loopback-sdk-ios/issues/40

**Comment by Vincentjonssonqi** [comment]
I just read that the swagger codegen project has support for swift SDK generation. Is this something that you guys plan to incorporate? If not, is there some downside to using the swagger generator on iOS? As you guys probably have a lot of experience with swagger It would be great to get your input on this matter. Is it a good idea and if so how do we start?

**Comment by hideya** [comment]
@Vincentjonssonqi , thanks for the info!  I didn't realize that  the swagger codegen has support for swift.  @raymondfeng @bajtos ,  would you give us any input on this?

**Comment by bajtos** [comment]
I think we need somebody to check what's the output of swagger codegen and see how much information was lost along the way, i.e. how much value can a  custom LoopBack codegen add.

**Comment by hideya** [comment]
@Vincentjonssonqi, would you give me a pointer to the description on the swagger codegen project's swift SDK generation support? (I've searched but couldn't find)

@bajtos, I've added a new Issue https://github.com/strongloop/loopback-sdk-ios-codegen/issues/3 for the discussion.

**Comment by kgoedecke** [comment]
@hideya Did you find out anything on that, I'm also quite curious what the generated SDK might look like.

**Comment by hideya** [comment]
@kgoedecke no, I haven't found anything about the swagger codegen's support for swift yet.  FYI, https://github.com/strongloop/loopback-sdk-ios-codegen

**Comment by kgoedecke** [comment]
@hideya interesting... I think swagger.io is the way to go on the long run, for now I guess we have to stick to the regular SDK.

**Comment by hideya** [comment]
@kgoedecke I was a bit blind -- https://github.com/swagger-api/swagger-codegen does have SwiftCodegen.java.  Yeah, I agree it is the way to go.  I should take a look into it.
